src/ucx_van.h

```diff
@@ -755,14 +755,12 @@ class UCXContext {
 
 class UCXVan : public Van {
  public:
-  UCXVan() {
-    setenv("UCX_USE_MT_MUTEX", "y", 0);
-    setenv("UCX_IB_NUM_PATHS", "2", 0);
-    setenv("UCX_SOCKADDR_CM_ENABLE", "y", 0);
-    setenv("UCX_RNDV_THRESH", "8k", 0);
```

Comment:
can we try to set them in a single thread (maybe somewhere in general van) for now?
We are going to support configuration files in UCX soon (see https://github.com/openucx/ucx/pull/6475)